src/detect-replace.c

```diff
@@ -276,15 +276,11 @@ int DetectReplaceLongPatternMatchTest(uint8_t *raw_eth_pkt, uint16_t pktsize,
     DecodeEthernet(&th_v, &dtv, p, GET_PKT_DATA(p), pktsize);
 
```

Comment:
a few if statements in the test have printfs and debug prints. I'm not sure how I should deal with then, any suggestions

Reply:
It's ok to leave them there, thanks for asking :)